### ISSUE TITLE
Error on arrow functions

### ISSUE THREAD

**Issue by sbowler** [Kocal/jsdoc-vuejs#78]
It looks like the latest fixes are failing on arrow functions.

```
\node_modules\jsdoc-vuejs\index.js:50
        if (e.doclet.memberof.endsWith('.methods')) {
                              ^

TypeError: Cannot read property 'endsWith' of undefined
```
This happens in an index.vue file with
```
export default {
  name: 'AuthModule',
  computed: {
    ...mapState({
      fooList: state => state.$_foo.fooList,
      barList: state => state.$_foo.barList
    })
  },
  created() {
...
```

I'm still trying to figure out how to work around this issue, but I thought I should let you know.

**Comment by sbowler** [comment]
I was able to work around this by adding a memberOf tag before each function, but ideally these should just be ignored.
```
export default {
  name: 'AuthModule',
  computed: {
    ...mapState({
      /**
       * @memberOf Auth
       */
      fooList: state => state.$_foo.fooList,
      /**
       * @memberOf Auth
       */
      barList: state => state.$_foo.barList
    })
  },
  created() {
...
```

**Comment by Kocal** [comment]
It has been fixed in #79 and published in 2.0.4.
